.github/workflows/e2e.yaml

```diff
@@ -13,12 +13,9 @@ jobs:
   examples:
     name: build examples
     runs-on: ubuntu-latest
-    container:
-      image: alpine:latest
-      options: |
-        --cap-add NET_ADMIN --cap-add SYS_ADMIN --security-opt seccomp=unconfined --security-opt apparmor:unconfined
```

Comment:
I'm curious why this is changing?  Should we switch it to Wolfi?

Reply:
I wanted to `docker run` the NTIA checker image, and `alpine:latest` doesn't have docker. Rather than trying to figure out how to make that work for Alpine or Wolfi, I just dropped out of the container.

AFAICT we were running inside `alpine:latest` as a holdover from the days where `apko` etc needed `apk` to be available, and beyond that it was just a convenience for installing `bubblewrap`, which is also just an `apt-get` away.